tests/test_anonfile.py

```diff
@@ -38,73 +39,116 @@ def remove_file(file: str) -> None:
     Path(file).unlink() if Path(file).exists() else None
 
 
-class TestAnonFileLibrary(unittest.TestCase):
+class MockData:
+    @staticmethod
+    def get_json_response(filename):
+        with open(filename, "r", encoding="utf-8") as json_file:
+            json_response = Mock(spec=Response)
+            json_response.__enter__ = MagicMock(return_value=json_response)
+            json_response.__exit__ = MagicMock()
+            json_response.status_code = 200
+            json_response.json.return_value = json.loads(json_file.read())
```

Comment:
This is okay and we can leave it at that!

Though in my opinion this adds another I/O operation that isn't transparent about the contract. It would be better to return a Python `dict`, e.g.

```python
json_response.json.return_value = {
  "status": 200,
  "file" : {
    "url" : { "etc." }
  }
}
```

which would eliminate the necessity of keeping the hard-to-read `preview.json` file (and in the same vein, the `upload.json` file)